littlepay/commands/groups.py

```diff
@@ -48,6 +48,9 @@ def groups(args: Namespace = None) -> int:
     elif command == "migrate":
         for group in groups:
             return_code += migrate_group(client, group.id, getattr(args, "force", False))
+    elif command == "funding_sources":
```

Comment:
I get why this was added as an `elif` condition here, but I think it makes the output a little strange.

First we see the list of funding sources:

```bash
$ littlepay groups funding_sources
Listing funding sources for: qa, mst [group...]
GroupFundingSourceResponse...
GroupFundingSourceResponse...
```

Then we see output about the matching group(s):

```bash
👥 Matching groups (N): qa, mst
GroupResponse...
```

Can we have the output work similar to e.g. `products`, where the group message comes first, then a single group is listed, then its products are listed, then the next group and its products, and so on...

```bash
$ littlepay groups products
👥 Matching groups (N): qa, mst
GroupResponse...
🛒 Linked products (N)
ProductResponse...
ProductResponse...
GroupResponse...
🛒 Linked products (N)
ProductResponse...
ProductResponse...
...
....
```

Reply:
Yep, this makes sense, I'll change the output to make it work similar to `products`.